.travis.yml

```diff
@@ -1,17 +0,0 @@
-language: node_js
-node_js:
-  - 6
-  - 8
-  - 10
-  - 12
-sudo: false
-before_install:
-  - export "PATH=./node_modules/.bin:$PATH"
-install: npm install
-script:
-  - gulp lint
-  - gulp dist
-  - gulp test karma
-after_success:
-  - gulp coveralls
```

Comment:
I've not included `coveralls` in the GitHub version, as this seems to error out on an invalid file. Likely this broke at some point, but was not caught as CI was broken. We'll have to look at this at a later point to get it fixed.